static/admin/js/pages/analytics/analytics-main.js

```diff
@@ -116,6 +116,7 @@ define(function(require) {
             'Last Month': [moment().subtract(1, 'month').startOf('month'), moment().subtract(1, 'month').endOf('month')],
             'This Year': [moment().startOf('year'), moment().endOf('year')],
             'Last Year': [moment().subtract(1, 'year').startOf('year'), moment().subtract(1, 'year').endOf('year')],
```

Comment:
Do we want to keep calendar years as the main options still? I think it would be better to do "This Fiscal Year", "Last Fiscal Year", etc

Reply:
I think we still want to keep the calendar year. I can double check if Francisco wants to remove any options or not. 
And yeah, "This fiscal year" sounds better. Will make the fix.